content/docs/reference/config/policy-and-telemetry/attribute-vocabulary/index.md

```diff
@@ -97,6 +97,12 @@ deployments will have agents (Envoy or Mixer adapters) that produce these attrib
 | `check.cache_hit` | boolean | Indicates whether Mixer check call hits local cache. | |
 | `quota.cache_hit` | boolean | Indicates whether Mixer quota call hits local cache. | |
 
+## Operate timestamp and duration attributes
+
+When operating with timestamp attributes, you can use `timestamp` funciton defined in [CEXL](/docs/reference/config/policies-and-telemetry/#expression-language) to construct expression, which converts a textual timestamp in RFC 3339 format into the `TIMESTAMP` type. Here are several example expressions: `request.time | timestamp("2018-01-01T22:08:41+00:00")`, `response.time > timestamp(2020-02-29T00:00:00-08:00)`. When using mixc to make calls into mixer, timestamp attributes should also be passed as texture timestamp in RFC 3339 format.
+
+When operating with duration attributes, you could directly using text format duration value to construct expression. A duration string is a possibly sequence of decimal numbers, each with optional fraction and a unit suffix, such as `300ms`, `1.5h` or `2h45m`. Here are several example expressions: `response.duration | "1m"`.
```

Comment:
Let's not document this behavior. Don't we have duration() function? If not, can we _please please_ add it. This behavior is very problematic since the language does not accept literal "1m" string.

Reply:
No, I don't think we have duration function. I removed this paragraph.